.github/workflows/ci.yml

```
@@ -29,8 +29,14 @@ jobs:
         if: startsWith(matrix.rust, 'nightly')
         run: cargo check -Z features=dev_dep
       - run: cargo test
+      - run: cargo test --features portable-atomic
       - name: Install cargo-hack
         uses: taiki-e/install-action@cargo-hack
+      - name: Run with Loom enabled
+        run: cargo check
```

```suggestion
        run: cargo test --test loom --release
```

We may want to create a new job for loom tests because loom tests tend to take a long time.